doc/introduction/photonic_hardware.rst

```diff
@@ -46,39 +46,23 @@ will execute.
 
 Jobs can also be submitted using the **non-blocking** :meth:`eng.run_async() <.RemoteEngine.run_async>`
 method. Unlike :meth:`eng.run() <.RemoteEngine.run>`, which returns a :class:`~.Results` object once the computation is
-complete, this method instead returns a :class:`~.Job` object directly that can be queried
+complete, this method instead returns a `xcc.Job` object directly that can be queried
 to get the job's status.
 
 >>> job = engine.run_async(prog, shots=3)
 >>> job.status
 "queued"
->>> job.refresh()
+>>> job.clear()
```

Comment:
To avoid race conditions:
```suggestion
>>> job.wait()
```

Reply:
I was thinking a bit about this, and thought that `job.clear()` was most similar to `job.refresh()` (keeping the job non-blocked). But I agree this seems good also.